fe/fe-core/src/main/java/org/apache/doris/analysis/AlterTableStmt.java

```diff
@@ -70,6 +76,50 @@ public void analyze(Analyzer analyzer) throws UserException {
         }
     }
 
+    public void rewriteAlterClause(OlapTable table) throws UserException {
+        List<AlterClause> clauses = new ArrayList<>();
+        for (AlterClause alterClause : ops) {
+            if (alterClause instanceof EnableFeatureClause
+                    && ((EnableFeatureClause) alterClause).getFeature() == EnableFeatureClause.Features.BATCH_DELETE) {
+                if (table.getKeysType() != KeysType.UNIQUE_KEYS) {
+                    throw new AnalysisException("Batch delete only support in unique tables.");
+                }
+                // has rollup table
+                if (table.getVisibleIndex().size() > 1) {
+                    for (MaterializedIndex idx : table.getVisibleIndex()) {
+                        if (idx.getId() == table.getBaseIndexId()) {
```

Comment:
miss to modify the base index?

Reply:
add  a column to rollup index it will add to base table automaticlly, if add a column here it will duplicated